apps/console/src/public/deployment.config.json

```diff
@@ -660,10 +660,10 @@
                 }
             },
             "identityVerificationProviders": {
-                "enabled": false,
+                "enabled": true,
                 "scopes": {
                     "create": [
-                        "internal_idvp_create"
+                        "internal_idvp_add"
```

Comment:
Let's update the framework as well.

Reply:
Correct scope is already defined in the framework. Issue is only in identity-apps